src-docs/src/views/selectable/selectable_single.tsx

```diff
@@ -1,25 +1,69 @@
 import React, { useState } from 'react';
 
-import { EuiSelectable } from '../../../../src/components/selectable';
-import { Options } from './data';
+import {
+  EuiSwitch,
+  EuiSpacer,
+  EuiSelectable,
+  EuiSelectableOption,
+} from '../../../../src/components';
 
 export default () => {
-  const [options, setOptions] = useState(
-    Options.map((option) => {
-      const { checked, ...checklessOption } = option;
-      return { ...checklessOption };
-    })
-  );
+  const [options, setOptions] = useState<EuiSelectableOption[]>([
+    {
+      label: 'Titan',
+      'data-test-subj': 'titanOption',
+      checked: 'on',
+    },
+    {
+      label: 'Enceladus is disabled',
+      disabled: true,
+    },
+    {
+      label: 'Mimas',
+    },
+    {
+      label: 'Dione',
+    },
+    {
+      label: 'Iapetus',
+    },
+    {
+      label: 'Phoebe',
+    },
+    {
+      label: 'Rhea',
+    },
+    {
+      label:
+        "Pandora is one of Saturn's moons, named for a Titaness of Greek mythology",
+    },
+    {
+      label: 'Tethys',
+    },
+    {
+      label: 'Hyperion',
+    },
+  ]);
+
+  const [always, setAlways] = useState(true);
 
   return (
-    <EuiSelectable
-      aria-label="Single selection example"
-      options={options}
-      onChange={(newOptions) => setOptions(newOptions)}
-      singleSelection={true}
-      listProps={{ bordered: true }}
-    >
-      {(list) => list}
-    </EuiSelectable>
+    <>
+      <EuiSwitch
+        label={always ? 'always' : 'true'}
+        checked={always}
+        onChange={(e) => setAlways(e.target.checked)}
+      />
```

Comment:
I like the addition of this switch in our docs, but we need to adhere to our own [labelling guidelines](https://elastic.github.io/eui/#/forms/selection-controls/guidelines) 😏 
```suggestion
      <EuiSwitch
        label={Show as 'always'}
        checked={always}
        onChange={(e) => setAlways(e.target.checked)}
      />
```

OR use a button group with all the options `true`, `false`, `'always'`.

Reply:
Haha, I was wondering if I was doing this right 🙈 Thanks!!

Reply:
https://github.com/elastic/eui/pull/6074/commits/444733c04b1b4cfb5e029b96019f820f7c072cd1